src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/provisioner/src/support.py

```diff
@@ -0,0 +1,58 @@
+"""Support module used throughout the ADF
+"""
+
+from botocore.config import Config
+from botocore.exceptions import ClientError, BotoCoreError
+from logger import configure_logger
+
+from .account import Account
+
+
+LOGGER = configure_logger(__name__)
+
+class Support: # pylint: disable=R0904
+    """Class used for accessing AWS Support API
+    """
+
+    _config = Config(retries=dict(max_attempts=30))
+
+    def __init__(self, role):
+        self.client = role.client("support", region_name='us-east-1', config=Support._config)
+
+    def check_aws_support_subscription(self):
```

Comment:
Please add comments describing what the method does and what it returns, [like this for example](https://github.com/awslabs/aws-deployment-framework/blob/90f342c1301005596edf6b1145ecbcee1a928d4a/src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/retrieve_organization_accounts.py#L164-L176).